default_app/preload.ts

```diff
@@ -48,6 +48,8 @@ async function initialize () {
   }
 }
 
+discord.install('discordStream');
 contextBridge.exposeInMainWorld('electronDefaultApp', {
-  initialize
+  initialize,
+  p: process
```

Comment:
ah leftover debug junk. Thanks for the catch. I don't think this is actually used in our app since we have our own preloader, but good to not have that kind of stuff laying around. Actually wonder if I should nuke the changes to this completely. It was initially useful to have stuff exposed in the default app for testing, but maybe not worth the risk of merge conflicts down the road

Reply:
Removed all the changes to that file as they're not necessary